### ISSUE TITLE
Error when pressing previous track key from a searched song.

### ISSUE THREAD

**Issue by a-singer** [InfiArtt/accessify-play#23]
Let me first say that it is very helpful that the add-on puts its issues into the NVDA log, it makes reporting far easier to be able to copy and paste. To reproduce

1.	Search for a song.
2.	Press enter and start playing one of the results. You will hear the music from your connected device, whatever it is.
3.	Press the previous track key (I haven't changed it from the nvda+shift+alt+left). I hear "command failed" and get the following in the log.

ERROR - external:globalPlugins.accesifyPlay.spotify_client.SpotifyClient._execute (19:40:41.045) - Thread-7383 (run) (11172):
Spotify command failed: http status: 403, code: -1 - https://api.spotify.com/v1/me/player/previous?device_id=828d3fea9bd2fabf3f545ae9a0f721e9886e6105:
 Player command failed: Restriction violated, reason: UNKNOWN
Traceback (most recent call last):
  File "C:\Users\test\AppData\Roaming\nvda\addons\AccessifyPlay\lib\spotipy\client.py", line 281, in _internal_call
    response.raise_for_status()
  File "requests\models.pyc", line 1024, in raise_for_status
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.spotify.com/v1/me/player/previous?device_id=828d3fea9bd2fabf3f545ae9a0f721e9886e6105

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\test\AppData\Roaming\nvda\addons\AccessifyPlay\globalPlugins\accesifyPlay\spotify_client.py", line 153, in _execute
    result = command(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\test\AppData\Roaming\nvda\addons\AccessifyPlay\lib\spotipy\client.py", line 1900, in previous_track
    return self._post(self._append_device_id("me/player/previous", device_id))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\test\AppData\Roaming\nvda\addons\AccessifyPlay\lib\spotipy\client.py", line 335, in _post
    return self._internal_call("POST", url, payload, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\test\AppData\Roaming\nvda\addons\AccessifyPlay\lib\spotipy\client.py", line 303, in _internal_call
    raise SpotifyException(
spotipy.exceptions.SpotifyException: http status: 403, code: -1 - https://api.spotify.com/v1/me/player/previous?device_id=828d3fea9bd2fabf3f545ae9a0f721e9886e6105:
 Player command failed: Restriction violated, reason: UNKNOWN 

I'm not sure what's up here. Shouldn't it go to the previous song I was playing? I thought it did but don't have anything else which uses the API, so am not at all sure.
